syntax = 'proto3';

message PlayerMove {
    float current_x = 1;
    float current_y = 2;
    float velocity_x = 3;
    float velocity_y = 4;
}

message OtherPlayerMove {
    uint32 player_id = 1;
    float starting_x = 2;
    float starting_y = 3;
    float velocity_x = 4;
    float velocity_y = 5;
}

message PlayerStop {
    float stopped_x = 1;
    float stopped_y = 2;
}

message OtherPlayerStop {
    uint32 player_id = 1;
    float stopped_x = 2;
    float stopped_y = 3;
}
